WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -95,6 +95,8 @@ - (BOOL)hidesBottomBarWhenPushed
 
 - (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
 {
+    [super willAnimateRotationToInterfaceOrientation:toInterfaceOrientation duration:duration];
```

Comment:
I was going to but then that's a whole other PR since all of the layout logic here is manual math.